ci/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM tpm2software/tpm2-tss:ubuntu-18.04
```

Comment:
We could really remove some code duplication by publishing this image 🤡

Reply:
Yes 🤡 Aren't we duplicating this image 3 times now?